src/Makefile.am

```diff
@@ -43,10 +43,14 @@ libxccl_la_SOURCES =       \
 	core/xccl_context.c    \
 	core/xccl_team.c       \
 	core/xccl_collective.c \
+	core/tasks_pool.c      \
```

Comment:
lets use "xccl_" prefix for files here to be consistent.

Reply:
fixed